sound/soc/sof/ipc4-topology.c

```diff
@@ -1485,6 +1485,87 @@ static int sof_ipc4_widget_free(struct snd_sof_dev *sdev, struct snd_sof_widget
 	return ret;
 }
 
+static int sof_ipc4_get_queue_id(struct snd_sof_widget *src_widget,
```

Comment:
commit message: I could not understand this paragraph

If widget has more than one sink/source pins, widget pin
binding array defined in topology is first checked for
queue ID allocation, error will be returned if cannot
allocate queue ID from pin binding array.

?? what does 'checked for queue ID allocation"?

Reply:
I intend to state the steps for queue id allocation, I will make it clearer.